pkgs/development/python-modules/jedi-language-server/default.nix

```diff
@@ -16,16 +16,16 @@
 
 buildPythonPackage rec {
   pname = "jedi-language-server";
-  version = "0.41.0";
+  version = "0.41.2-dev";
```

Comment:
can we use unreleased version?

Reply:
Mark this as `0.41.x-unstable-xxxx-xx-xx`.